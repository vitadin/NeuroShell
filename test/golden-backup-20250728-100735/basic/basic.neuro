%% Basic functionality test
\set[_echo_command="true"]
\set[name="test"]
\set[status="working"]
\get[name]
\get[status]