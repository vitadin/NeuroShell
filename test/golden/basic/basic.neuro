%% Basic functionality test
\set[_echo_commands="true"]
\set[name="test"]
\set[status="working"]
\get[name]
\get[status]