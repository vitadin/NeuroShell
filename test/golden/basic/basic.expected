Setting _echo_commands = true
%%> \set[name="test"]
Setting name = test
%%> \set[status="working"]
Setting status = working
%%> \get[name]
name = test
%%> \get[status]
status = working